keyboards/monsgeek/m2/info.json

```diff
@@ -0,0 +1,263 @@
+            {"matrix": [2, 19], "x": 210, "y": 25, "flags": 4},
+            {"matrix": [2, 20], "x": 224, "y": 25, "flags": 4},
+            {"matrix": [3, 0], "x": 0, "y": 38, "flags": 4},
+            {"matrix": [3, 1], "x": 13, "y": 38, "flags": 4},
+            {"matrix": [3, 2], "x": 26, "y": 38, "flags": 4},
+            {"matrix": [3, 3], "x": 39, "y": 38, "flags": 4},
+            {"matrix": [3, 4], "x": 52, "y": 38, "flags": 4},
+            {"matrix": [3, 5], "x": 65, "y": 38, "flags": 4},
+            {"matrix": [3, 6], "x": 79, "y": 38, "flags": 4},
+            {"matrix": [3, 7], "x": 92, "y": 38, "flags": 4},
+            {"matrix": [3, 8], "x": 105, "y": 38, "flags": 4},
+            {"matrix": [3, 9], "x": 118, "y": 38, "flags": 4},
+            {"matrix": [3, 10], "x": 131, "y": 38, "flags": 4},
+            {"matrix": [3, 11], "x": 144, "y": 38, "flags": 4},
+            {"matrix": [3, 13], "x": 163, "y": 38, "flags": 4},
+            {"matrix": [3, 17], "x": 184, "y": 38, "flags": 4},
+            {"matrix": [3, 18], "x": 197, "y": 38, "flags": 4},
+            {"matrix": [3, 19], "x": 210, "y": 38, "flags": 4},
+            {"matrix": [4, 0], "x": 0, "y": 51, "flags": 4},
+            {"matrix": [4, 1], "x": 26, "y": 51, "flags": 4},
+            {"matrix": [4, 2], "x": 39, "y": 51, "flags": 4},
+            {"matrix": [4, 3], "x": 52, "y": 51, "flags": 4},
+            {"matrix": [4, 4], "x": 65, "y": 51, "flags": 4},
+            {"matrix": [4, 5], "x": 79, "y": 51, "flags": 4},
+            {"matrix": [4, 6], "x": 92, "y": 51, "flags": 4},
+            {"matrix": [4, 7], "x": 105, "y": 51, "flags": 4},
+            {"matrix": [4, 8], "x": 118, "y": 51, "flags": 4},
+            {"matrix": [4, 9], "x": 131, "y": 51, "flags": 4},
+            {"matrix": [4, 10], "x": 144, "y": 51, "flags": 4},
+            {"matrix": [4, 13], "x": 158, "y": 51, "flags": 4},
+            {"matrix": [4, 15], "x": 171, "y": 51, "flags": 4},
+            {"matrix": [4, 17], "x": 184, "y": 51, "flags": 4},
+            {"matrix": [4, 18], "x": 197, "y": 51, "flags": 4},
+            {"matrix": [4, 19], "x": 210, "y": 51, "flags": 4},
+            {"matrix": [4, 20], "x": 224, "y": 51, "flags": 4},
+            {"matrix": [5, 0], "x": 0, "y": 64, "flags": 4},
+            {"matrix": [5, 1], "x": 18, "y": 64, "flags": 4},
+            {"matrix": [5, 2], "x": 37, "y": 64, "flags": 4},
+            {"matrix": [5, 5], "x": 92, "y": 64, "flags": 4},
+            {"matrix": [5, 9], "x": 130, "y": 64, "flags": 4},
+            {"matrix": [5, 10], "x": 145, "y": 64, "flags": 4},
+            {"matrix": [5, 11], "x": 158, "y": 64, "flags": 4},
+            {"matrix": [5, 15], "x": 171, "y": 64, "flags": 4},
+            {"matrix": [5, 16], "x": 184, "y": 64, "flags": 4},
+            {"matrix": [5, 18], "x": 197, "y": 64, "flags": 4},
+            {"matrix": [5, 19], "x": 210, "y": 64, "flags": 4}
```

Comment:
In the case I create a custom keymap, and chose to enable the `RGB_MATRIX_ALPHAS_MODS` animation, and then compile from repo... it would not work as **all** LED's are indexed as `LED_FLAG_KEYLIGHT`.  Should some of these not be `LED_FLAG_MODIFIER`?